pkgs/desktops/xfce/core/gtk-xfce-engine.nix

```diff
@@ -1,17 +1,19 @@
-{ stdenv, fetchurl, pkgconfig, intltool, gtk }:
+{ stdenv, fetchurl, pkgconfig, intltool, gtk, withGtk3 ? false, gtk3 }:
```

Comment:
Maybe we can have both `gtk_xfce_engine` with support for GTK+2 and `gtk3_xfce_engine` with support for GTK+3. This would be similar to what [Gentoo Linux](https://packages.gentoo.org/packages/x11-themes/gtk-engines-xfce) offer.

In a similar fashion we have in the repository `libxfce4ui` and `libxfce4ui_gtk3`, and `xfce4panel` and `xfce4panel_gtk3`.

@edolstra do you think this is plausible?